xml/cha_images.xml

```diff
@@ -156,54 +163,87 @@
      <row>
       <entry>
        <para>
-        Instances are ready to use
+        Customer
```

Comment:
How about heading = “customer relationship” and “with cloud provider” for payg and “with cloud provider and SUSE” for byos?

Reply:
Thanks, applied in latest commit.